src/app/page.tsx

```diff
@@ -1,11 +1,33 @@
 import React from 'react';
 
+import Input from '@/components/input/Input';
 import ZustandTest from '@/components/ZustandTest';
 
 const page = () => {
   return (
     <div>
       <ZustandTest />
+      <div className='my-4 grid gap-4'>
+        <Input size='small' placeholder='size : small' />
+        <Input size='large' placeholder='size : large' />
+        <label htmlFor='error'>error</label>
+        <div>
+          <Input
+            id='error'
```

Comment:
```suggestion
const page = () => {
    const errorId = useId() // 상위에서 useId선언
...
            id={errorId + 'error'}
```
react 에서 하드코딩 id는 최대한 사용하지 않도록 자제해야 해요.
지금은 페이지 단위라서 문제가 없어 보이지만 해당 기능이 component화 된다면 DOM에서 중복된 id로 예상하지 못한 충돌이 발생할 수 있어요!

`htmlFor` 속성으로 인해 부득이하게 사용해야한다면 `useId hook` 을 사용하는걸 권장드려요 😉

[useId hook이란?](https://react.dev/reference/react/useId)

Reply:
감사합니다아아!!! useId 처음 들어봤어요..! 다음 작업부터는 useId 사용할게여!